pkgs/by-name/cl/cloudflare-warp/package.nix

```diff
@@ -90,6 +101,8 @@ stdenv.mkDerivation rec {
     wrapProgram $out/bin/warp-svc --prefix PATH : ${lib.makeBinPath [ nftables ]}
   '';
 
+  passthru.tests.version = testers.testVersion { package = cloudflare-warp; };
```

Comment:
I'd suggest `versionCheckHook`, as this is not very likely be discovered (since ofborg doesn't run them).

Reply:
Thank you for letting me know! I didn't notice this recommendation and the documentation updates f42d4debf076b03f48b24fb59a7a75d0d8c29363 

And thank you for pointing out the bot behavior regarding unfree packages 👍